### PR TITLE
tests: ghak99 EXECVE blank args missing

### PR DESCRIPTION
**Pull request by rgbriggs**
Test for missing blank args in EXECVE record.

See: https://github.com/linux-audit/audit-kernel/issues/99
Signed-off-by: Richard Guy Briggs <rgb@redhat.com>

### CODE REVIEW

**Comment by rgbriggs** [comment]
On 2018-11-20 19:27, Paul Moore wrote:
> I would rather see a test for blank args included in the existing exec_execve test than to add a new execve test.  I think if you wanted to modify the tests/exec_execve test to add a "count_blanks" test type that would be okay.

I seriously considered that, but creating a new regression test was significantly simpler and much easier to associate with a specific bug.

**Comment by pcmoore** [comment]
Let me rephrase my earlier comment - if you would like to see a test like this added to the test suite, modify the existing tests/exec_execve test to add a new "count_blanks" test type to verify mixing blank and non-blank arguments.

**Comment by pcmoore** [comment]
> Let me rephrase my earlier comment - if you would like to see a test like this added to the test suite, modify the existing tests/exec_execve test to add a new "count_blanks" test type to verify mixing blank and non-blank arguments.

With the comment above in mind, let's close out this PR (and by extension this approach) and create a new PR that extends the existing tests/exec_execve test.

**Comment by rgbriggs** [comment]
On 2018-12-12 09:35, Paul Moore wrote:
> > Let me rephrase my earlier comment - if you would like to see a test like this added to the test suite, modify the existing tests/exec_execve test to add a new "count_blanks" test type to verify mixing blank and non-blank arguments.
> 
> With the comment above in mind, let's close out this PR (and by extension this approach) and create a new PR that extends the existing tests/exec_execve test.

Please go ahead.  I already looked at the exec_execve test and at this point I don't intend to try to add this test to it.
